src/Pretzel.Tests/Extensibility/Extensions/VirtualDirectorySupportTests.cs

```diff
@@ -52,21 +50,23 @@ public void processes_only_when_vdir_argument_is_passed()
             // arrange
             var returnThis = Substitute.For<FileBase>();
             fileSystem.File.Returns(returnThis);
-            var optionSet = new OptionSet();
-            vdirSupport.UpdateOptions(optionSet);
-            optionSet.Parse(new[] { "--vDir", "something" });
```

Comment:
Sorry if I have missed it, but there is no more parsing test?

Reply:
Added `AzureHostSupportArgumentsTests` and `VirtualDirectorySupportArgumentsTests` and found 1 bug 😀 and fixed it